app/assets/stylesheets/shared/_markdown.scss

```diff
@@ -0,0 +1,20 @@
+.markdown {
+  h1 {}
+  h2 {}
+  h3 {}
+  h4 {}
+  h5 {}
+  p {}
+  em {}
+  strong {}
+  u {}
+  mark {}
+  q {}
+  blockquote {}
+  ul {
+    li {}
```

Comment:
Empty rule<br>Selector should have depth of applicability no greater than 2, but was 3